test/inductor/test_aot_inductor.py

```diff
@@ -6726,6 +6726,27 @@ def forward(self, x):
             torch._inductor.aot_compile(ep.module(), inputs)
         self.assertEqual([r.msg == "create_env" for r in records].count(True), 1)
 
+    @make_logging_test(dynamic=logging.DEBUG)
+    def test_shape_env_reuse_zero_consts_use_consts_asm_false(self, records):
+        saved_cfg: bool = torch._inductor.config.aot_inductor.use_consts_asm_build
+        torch._inductor.config.aot_inductor.use_consts_asm_build = False
```

Comment:
We should use config.patch for this, like https://github.com/pytorch/pytorch/blob/e7cc42df58a86bee05944f6e80c535aa1d099443/test/inductor/test_aot_inductor.py#L273